Containerfile

```diff
@@ -71,7 +71,7 @@ RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install \
 # Remove gnutls, libarchive and everything that depends on it. 
 # https://github.com/skupperproject/skupper-router/issues/1477
 # https://github.com/skupperproject/skupper-router/issues/1639
-RUN microdnf -y remove gnutls glib2 gobject-introspection libpeas microdnf gnupg2 gpgme libdnf json-glib libmodulemd librepo librhsm libsolv rpm rpm-libs libarchive libyaml libusbx systemd-libs
+RUN microdnf -y remove gettext-libs gettext libxml2 gnutls glib2 gobject-introspection libpeas microdnf gnupg2 gpgme libdnf json-glib libmodulemd librepo librhsm libsolv rpm rpm-libs libarchive libyaml libusbx systemd-libs
```

Comment:
If `gettext` isn’t needed, wouldn’t it be better to skip installing in line 67, instead of installing it and then removing it?

Reply:
> wouldn’t it be better to skip installing it in line 67

Yes, agree. Nice catch. I have fixed it.